s4e-web/src/app/views/settings/settings.component.html

```diff
@@ -26,7 +26,11 @@
   <nav
     class="navigation"
   >
-    <a routerLink="/settings">
+    <a [routerLink]="
+      (hasAnyAdminInstitution$ | async)
+        ? (isAdminOfOneInstitution$ | async) ? '/settings/dashboard' : '/settings/institutions'
+        : '/settings'
+    ">
```

Comment:
This is a little to much nested logic in `routerLink` it might be better to make stream in component and only pipe it with `async` here.

Reply:
I was considering it, now I'm sure that I should do it from beginning :P